src/components/v5/common/ActionSidebar/partials/Motions/steps/StakingStep/StakingStep.tsx

```diff
@@ -64,6 +67,24 @@ const StakingStep: FC<StakingStepProps> = ({ className, isActive }) => {
     objectingStakesPercentageValue !== 100 &&
     supportingStakesPercentageValue === 100;
 
+  useEffect(() => {
```

Comment:
could this maybe be reworked into a `useMemo` instead of `useEffect` + `useState`? Not sure if applicable, but since we don't call `setCardTitleText` anywhere else, it's good practice to just memo this variable

Reply:
@bassgeta I've used useMemo here